api/spec/mutations/sign_in_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+module Mutations
+   module Auth
```

Comment:
Since all `spec` files are named according to what they do, there is no need to have them inside any modules.